### PR TITLE
Copy labels from parent PR to backport PR(s)

### LINKED ISSUE #26: Copy over labels to backport PRs
When backporting, I try to copy over labels from the original PR to the backport PR, for example the "trivial" or "documentation" labels.

Perhaps this is something that can be automated and done by bedevere.
At the very least, the "trivial" label should be copied over.

UPDATE:
Copy over `skip issue` label instead of `trivial`. `trivial` label is no more.

UPDATE 2.0:
At the very least, copy over `skip issue` and `skip news` labels.

### PR DESCRIPTION
**Pull request by brettcannon**
Closes #26

### CODE REVIEW

File: bedevere/backport.py [hunk 1]
```diff
@@ -15,23 +15,46 @@
                     '[{branch} branch](https://github.com/python/cpython/tree/{branch}).')
 
 
+async def _copy_over_labels(gh, original_issue, backport_issue):
+    """Copy over relevent labels from the origial PR to the backport PR."""
```
Comment (Mariatta) [inline bedevere/backport.py]: origial -> original

File: bedevere/backport.py [hunk 2]
```diff
@@ -15,23 +15,46 @@
...
 
-    issue_to_backport = await gh.getitem(event.data['repository']['issues_url'],
-                                         {'number': pr_to_backport})
-    if backport_label not in util.labels(issue_to_backport):
-        return
-    await gh.delete(issue_to_backport['labels_url'], {'name': backport_label})
-    message = MESSAGE_TEMPLATE.format(branch=branch, pr=backported_pr)
-    await gh.post(issue_to_backport['comments_url'], data={'body': message})
+    original_issue = await gh.getitem(event.data['repository']['issues_url'],
+                                         {'number': original_pr_number})
```
Comment (Mariatta) [inline bedevere/backport.py]: Should the `{` line up with the `(` from the previous line?

**Comment by Mariatta** [comment]
Thanks! This looks good :)

File: bedevere/backport.py (hunk 2 above)
Comment (brettcannon) [reply bedevere/backport.py]: Yep 😄

**Comment by Mariatta** [comment]
👍

**Pull request merged**